packages/delegation-controller/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export type {
+  DelegationControllerActions,
+  DelegationControllerEvents,
+  DelegationControllerMessenger,
+  DelegationEntry,
+  DelegationFilter,
+} from './types';
+
+export {
+  DelegationController,
+  getDefaultDelegationControllerState,
```

Comment:
Is this needed externally from the clients?

Reply:
Not that I know of, just exported because of [documentation](https://github.com/MetaMask/core/blob/main/docs/controller-guidelines.md#provide-a-default-representation-of-state). Should we drop this?

Reply:
As mentioned above, this will be dropped.